Clarified the use of the AIP client in coexistence with built-in labels (added paragraphs in the right order this time) (#6586)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/sensitivity-labels-office-apps.md

### Office built-in labeling client and the Azure Information Protection client

> You may want to use the built-in sensitivity labeling capabilities in Office to benefit from the latest features and enhance user experience while at the same time having the abilit to label and protect non-Office files from the desktop with the Azure Information Protection client. 

Reviewer: ```suggestion
You might want to use the built-in sensitivity labeling capabilities in Office to benefit from the latest features and enhance user experience while at the same time having the ability to label and protect non-Office files from the desktop with the Azure Information Protection client. 
```

> By default, if users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2) the built-in labeling client is turned off in their Office apps and the plugin functionality is used instead. 

Reviewer: ```suggestion
By default, if users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2), the built-in labeling client is turned off in their Office apps and the plug-in functionality is used instead. 
```

> To maintain the ability to use the Azure Information Protection clientfor labeling non-Office files from the desktop without disabling the built-in labeling functinality within Office apps, we recommend you use the Group Policy setting **List of managed add-ins** as documented in [No Add-ins loaded due to group policy settings for Office 2013 and Office 2016 programs](https://support.microsoft.com/help/2733070/no-add-ins-loaded-due-to-group-policy-settings-for-office-2013-and-off).

Reviewer: ```suggestion
To maintain the ability to use the Azure Information Protection client for labeling non-Office files from the desktop without disabling the built-in labeling functionality within Office apps, we recommend you use the Group Policy setting **List of managed add-ins** as documented in [No Add-ins loaded due to group policy settings for Office 2013 and Office 2016 programs](https://support.microsoft.com/help/2733070/no-add-ins-loaded-due-to-group-policy-settings-for-office-2013-and-off).
```

> [!NOTE]
> If you use the Group Policy setting **Use the Sensitivity feature in Office to apply and view sensitivity labels** and set this to **1**, there are some situations where the Azure Information Protection client might still load in Office apps. Blocking the add-in from loading in each app prevents this happening.

> You can also use the following registry value to quickly toggle between using the Azure Information Protection plugin and built-in sensitivity labels in Office apps:

Reviewer: ```suggestion
You can also use the following registry value to quickly toggle between using the Azure Information Protection plug-in and built-in sensitivity labels in Office apps:
```

|**Registry key**|**Type**|**Value** |
|:-------|:------:|--------:|
|HKCU\Software\Microsoft\Office\16.0\Common\Security\Labels\UseOfficeForLabelling  |REG_DWORD  |1  |

> Setting this value to 1 enables the built-in sensitivity labeling options in Office, setting it to 0 enables the AIP plugin if it is installed. 

Reviewer: ```suggestion
Setting this value to **1** enables the built-in sensitivity labeling options in Office; setting it to **0** enables the AIP plug-in if it is installed. 
```
